storops/unity/resource/lun.py

```diff
@@ -18,14 +18,163 @@
 import logging
 
 from storops.unity.resource import UnityResource, UnityResourceList
+import storops.unity.resource.pool
+from storops.unity.enums import TieringPolicyEnum, NodeEnum, HostLUNAccessEnum
+from storops.unity.resource.storage_resource import UnityStorageResource
+from storops.exception import UnityResourceNotFoundError
 
 __author__ = 'Jay Xu'
 
 log = logging.getLogger(__name__)
 
 
 class UnityLun(UnityResource):
-    pass
+    @classmethod
+    def create(cls, cli, name, pool, size, sp=None, host_access=None,

```

Comment:
Is `name` a required parameter for rest API?  If not , we could give a default value to save the caller's effort.